openmindat/countries.py

```diff
@@ -92,13 +96,40 @@ def save(self, FILE_NAME = ''):
                 None
 
             Example:
-                >>> cr = countriesRetriever()
-                >>> cr.save()
+                >>> cr = CountriesListRetriever()
+                >>> cr.page(2).save()
         '''
         file_name = FILE_NAME
         
         self.saveto('', file_name)
         
+    def get_object(self):
```

Comment:
I think I understand what you were talking about. Firstly, using the function name `get_object` is not recommended, as it can be very confusing for users to determine the object types.
As for your coding, I can tell there will be two returned types according to the difference in the querying param.
I would like you to commit the following revision:
1. change the function name to get_list for better readability
2. see the following review comment